[Bug]: Test test_figure.py::test_subfigure_ss[png] FAILED on ppc64le

### Bug summary

test_figure.py::test_subfigure_ss[png] FAILED on power and same test passed on x86 machine


### Code for reproduction

```python
pytest lib/matplotlib/tests/test_figure.py -v
```


### Actual outcome

bash-4.4# pytest lib/matplotlib/tests/test_figure.py -v
========================================================= test session starts =========================================================
platform linux -- Python 3.8.12, pytest-7.1.2, pluggy-1.0.0 -- /usr/bin/python3.8
cachedir: .pytest_cache
rootdir: /matplotlib, configfile: pytest.ini
collected 87 items

lib/matplotlib/tests/test_figure.py::test_align_labels[png] PASSED                                                              [  1%]
lib/matplotlib/tests/test_figure.py::test_align_labels[svg] SKIPPED (Cannot compare svg files on this system)                   [  2%]
lib/matplotlib/tests/test_figure.py::test_figure_label PASSED                                                                   [  3%]
lib/matplotlib/tests/test_figure.py::test_fignum_exists PASSED                                                                  [  4%]
lib/matplotlib/tests/test_figure.py::test_clf_keyword PASSED                                                                    [  5%]
lib/matplotlib/tests/test_figure.py::test_figure[png] PASSED                                                                    [  6%]
lib/matplotlib/tests/test_figure.py::test_figure[pdf] SKIPPED (Cannot compare pdf files on this system)                         [  8%]
lib/matplotlib/tests/test_figure.py::test_figure[svg] SKIPPED (Cannot compare svg files on this system)                         [  9%]
lib/matplotlib/tests/test_figure.py::test_figure_legend[png] PASSED                                                             [ 10%]
lib/matplotlib/tests/test_figure.py::test_figure_legend[pdf] SKIPPED (Cannot compare pdf files on this system)                  [ 11%]
lib/matplotlib/tests/test_figure.py::test_figure_legend[svg] SKIPPED (Cannot compare svg files on this system)                  [ 12%]
lib/matplotlib/tests/test_figure.py::test_gca PASSED                                                                            [ 13%]
lib/matplotlib/tests/test_figure.py::test_add_subplot_subclass PASSED                                                           [ 14%]
lib/matplotlib/tests/test_figure.py::test_add_subplot_invalid PASSED                                                            [ 16%]
lib/matplotlib/tests/test_figure.py::test_suptitle[png] PASSED                                                                  [ 17%]
lib/matplotlib/tests/test_figure.py::test_suptitle[pdf] SKIPPED (Cannot compare pdf files on this system)                       [ 18%]
lib/matplotlib/tests/test_figure.py::test_suptitle[svg] SKIPPED (Cannot compare svg files on this system)                       [ 19%]
lib/matplotlib/tests/test_figure.py::test_suptitle_fontproperties PASSED                                                        [ 20%]
lib/matplotlib/tests/test_figure.py::test_alpha[png] PASSED                                                                     [ 21%]
lib/matplotlib/tests/test_figure.py::test_alpha[svg] SKIPPED (Cannot compare svg files on this system)                          [ 22%]
lib/matplotlib/tests/test_figure.py::test_too_many_figures PASSED                                                               [ 24%]
lib/matplotlib/tests/test_figure.py::test_iterability_axes_argument PASSED                                                      [ 25%]
lib/matplotlib/tests/test_figure.py::test_set_fig_size PASSED                                                                   [ 26%]
lib/matplotlib/tests/test_figure.py::test_axes_remove PASSED                                                                    [ 27%]
lib/matplotlib/tests/test_figure.py::test_figaspect PASSED                                                                      [ 28%]
lib/matplotlib/tests/test_figure.py::test_autofmt_xdate[None] PASSED                                                            [ 29%]
lib/matplotlib/tests/test_figure.py::test_autofmt_xdate[both] PASSED                                                            [ 31%]
lib/matplotlib/tests/test_figure.py::test_autofmt_xdate[major] PASSED                                                           [ 32%]
lib/matplotlib/tests/test_figure.py::test_autofmt_xdate[minor] PASSED                                                           [ 33%]
lib/matplotlib/tests/test_figure.py::test_change_dpi PASSED                                                                     [ 34%]
lib/matplotlib/tests/test_figure.py::test_invalid_figure_size[1-nan] PASSED                                                     [ 35%]
lib/matplotlib/tests/test_figure.py::test_invalid_figure_size[-1-1] PASSED                                                      [ 36%]
lib/matplotlib/tests/test_figure.py::test_invalid_figure_size[inf-1] PASSED                                                     [ 37%]
lib/matplotlib/tests/test_figure.py::test_invalid_figure_add_axes PASSED                                                        [ 39%]
lib/matplotlib/tests/test_figure.py::test_subplots_shareax_loglabels PASSED                                                     [ 40%]
lib/matplotlib/tests/test_figure.py::test_savefig PASSED                                                                        [ 41%]
lib/matplotlib/tests/test_figure.py::test_savefig_warns PASSED                                                                  [ 42%]
lib/matplotlib/tests/test_figure.py::test_savefig_backend PASSED                                                                [ 43%]
lib/matplotlib/tests/test_figure.py::test_figure_repr PASSED                                                                    [ 44%]
lib/matplotlib/tests/test_figure.py::test_warn_cl_plus_tl PASSED                                                                [ 45%]
lib/matplotlib/tests/test_figure.py::test_add_artist[png] PASSED                                                                [ 47%]
lib/matplotlib/tests/test_figure.py::test_add_artist[pdf] SKIPPED (Don't know how to convert .pdf files to png)                 [ 48%]
lib/matplotlib/tests/test_figure.py::test_fspath[png] PASSED                                                                    [ 49%]
lib/matplotlib/tests/test_figure.py::test_fspath[pdf] PASSED                                                                    [ 50%]
lib/matplotlib/tests/test_figure.py::test_fspath[ps] PASSED                                                                     [ 51%]
lib/matplotlib/tests/test_figure.py::test_fspath[eps] PASSED                                                                    [ 52%]
lib/matplotlib/tests/test_figure.py::test_fspath[svg] PASSED                                                                    [ 54%]
lib/matplotlib/tests/test_figure.py::test_tightbbox PASSED                                                                      [ 55%]
lib/matplotlib/tests/test_figure.py::test_axes_removal PASSED                                                                   [ 56%]
lib/matplotlib/tests/test_figure.py::test_removed_axis PASSED                                                                   [ 57%]
lib/matplotlib/tests/test_figure.py::test_picking_does_not_stale PASSED                                                         [ 58%]
lib/matplotlib/tests/test_figure.py::test_add_subplot_twotuple PASSED                                                           [ 59%]
lib/matplotlib/tests/test_figure.py::test_tightbbox_box_aspect[svg] SKIPPED (Cannot compare svg files on this system)           [ 60%]
lib/matplotlib/tests/test_figure.py::test_animated_with_canvas_change[svg] SKIPPED (Don't know how to convert .svg files to...) [ 62%]
lib/matplotlib/tests/test_figure.py::test_animated_with_canvas_change[pdf] SKIPPED (Don't know how to convert .pdf files to...) [ 63%]
lib/matplotlib/tests/test_figure.py::test_animated_with_canvas_change[eps] SKIPPED (Don't know how to convert .eps files to...) [ 64%]
lib/matplotlib/tests/test_figure.py::test_animated_with_canvas_change[png] PASSED                                               [ 65%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_basic[x0-png] PASSED                                               [ 66%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_basic[x1-png] PASSED                                               [ 67%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_all_nested[png] PASSED                                             [ 68%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_nested[png] PASSED                                                 [ 70%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_nested_tuple[png] PASSED                                           [ 71%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_empty[x0-None-png] PASSED                                          [ 72%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_empty[x1-SKIP-png] PASSED                                          [ 73%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_empty[x2-0-png] PASSED                                             [ 74%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_empty[x3-None-png] PASSED                                          [ 75%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_empty[x4-SKIP-png] PASSED                                          [ 77%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_empty[x5-0-png] PASSED                                             [ 78%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_fail_list_of_str PASSED                                            [ 79%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_subplot_kw[subplot_kw0-png] PASSED                                 [ 80%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_subplot_kw[subplot_kw1-png] PASSED                                 [ 81%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_subplot_kw[None-png] PASSED                                        [ 82%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_string_parser PASSED                                               [ 83%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_single_str_input[AAA\nBBB-png] PASSED                              [ 85%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_single_str_input[\nAAA\nBBB\n-png] PASSED                          [ 86%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_single_str_input[ABC\nDEF-png] PASSED                              [ 87%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_fail[x0-(?m)we found that the label .A. specifies a non-rectangular or non-contiguous area.] PASSED [ 88%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_fail[x1-There are duplicate keys .* between the outer layout] PASSED [ 89%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_fail[AAA\nc\nBBB-All of the rows must be the same length] PASSED   [ 90%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_fail[x3-All of the rows must be the same length] PASSED            [ 91%]
lib/matplotlib/tests/test_figure.py::TestSubplotMosaic::test_hashable_keys[png] PASSED                                          [ 93%]
lib/matplotlib/tests/test_figure.py::test_reused_gridspec PASSED                                                                [ 94%]
lib/matplotlib/tests/test_figure.py::test_subfigure[png] PASSED                                                                 [ 95%]
lib/matplotlib/tests/test_figure.py::test_subfigure_ss[png] FAILED                                                              [ 96%]
lib/matplotlib/tests/test_figure.py::test_subfigure_double[png] PASSED                                                          [ 97%]
lib/matplotlib/tests/test_figure.py::test_add_subplot_kwargs PASSED                                                             [ 98%]
lib/matplotlib/tests/test_figure.py::test_add_axes_kwargs PASSED                                                                [100%]

============================================================== FAILURES ===============================================================
_______________________________________________________ test_subfigure_ss[png] ________________________________________________________

expected = '/matplotlib/result_images/test_figure/test_subfigure_ss-expected.png'
actual = PosixPath('/matplotlib/result_images/test_figure/test_subfigure_ss.png'), tol = 0

    def _raise_on_image_difference(expected, actual, tol):
        __tracebackhide__ = True

        err = compare_images(expected, actual, tol, in_decorator=True)
        if err:
            for key in ["actual", "expected", "diff"]:
                err[key] = os.path.relpath(err[key])
>           raise ImageComparisonFailure(
                ('images not close (RMS %(rms).3f):'
                    '\n\t%(actual)s\n\t%(expected)s\n\t%(diff)s') % err)
E           matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 0.012):
E               result_images/test_figure/test_subfigure_ss.png
E               result_images/test_figure/test_subfigure_ss-expected.png
E               result_images/test_figure/test_subfigure_ss-failed-diff.png

lib/matplotlib/testing/decorators.py:139: ImageComparisonFailure
======================================================= short test summary info =======================================================
FAILED lib/matplotlib/tests/test_figure.py::test_subfigure_ss[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images no...
============================================== 1 failed, 73 passed, 13 skipped in 18.09s ==============================================

### Expected outcome

All test should pass

### Additional information

All test passed on rhel x86_64 but failed on ppc64le architecture.

### Operating system

RHEL

### Matplotlib Version

v3.4.1

### Matplotlib Backend

agg

### Python version

python 3.8

### Jupyter version

NA

### Installation

git checkout

--- a/lib/matplotlib/tests/test_figure.py
+++ b/lib/matplotlib/tests/test_figure.py
@@ -1130,7 +1130,7 @@ def test_subfigure_dpi():
 
 
 @image_comparison(['test_subfigure_ss.png'], style='mpl20',
-                  savefig_kwarg={'facecolor': 'teal'})
+                  savefig_kwarg={'facecolor': 'teal'}, tol=0.02)
 def test_subfigure_ss():
     # test assigning the subfigure via subplotspec
     np.random.seed(19680801)

